sync/class.jetpack-sync-actions.php

```diff
@@ -441,6 +452,18 @@ static function get_sync_status() {
 			)
 		);
 	}
+
+	static function async_sender() {
+		$url = add_query_arg( 'jetpack_sync_async_sender', true, site_url() );
```

Comment:
Can we fake a login request to the wp-admin interface? 

I think it would be helpful for the callables that we sync. 

Because of how plugins hook into filters and such it would help with the to have a logged in wp-admin request depending on the request we might loose out in some info. 

Should we define our own more specific url that generated the callable in a particular way.